### DISCUSSION TITLE
"Something went wrong while loading short URLs" after update

### DISCUSSION THREAD

**Discussion by morphinestyle** [shlinkio/shlink#1951] [Help wanted] [answered]
### Shlink version

3.7.0

### PHP version

8.2

### How do you serve Shlink

Other (explain in summary)

### Database engine

MariaDB

### Database version

10.11

### Summary

I have successfully installed and tested version 6.4.0. (WOW!)

I specifically installed version 6.4.0 to test an upgrade before using the service in production.
I then unzipped the 3.7.0 dist file into a different folder, imported the old configuration and finally also activated tracking with Matomo :)

At this point I change the served folder as "public_html" from folder **shlink3.6.4** => to folder **shlink3.7.0**.

At this point the redirects still works correctly, and from the web client I also correctly display the "3.7.0" version, **unfortunately however the list of links is empty and I display the message: "Something went wrong while loading short URLs"**.

**What i missing?** I also generated new API, but it doesn't change the result.

**Comment by acelaya** [comment]
Frequently, people forget to set the right permissions to the `data` folder, which could cause this kind of error in some requests.

If this is not the case, there should be some logs inside `data/log` with more information.

**Reply by morphinestyle** [reply, marked as answer]
Everything was already correct Alejandro: I had to leave the computer for half an hour, when I returned everything worked correctly without doing anything.

In my specific case: restarting Litespeed (webserver) wasn't enough, I suppose I'll also have to restart other services (I'll have time to understand what they are).

**Now all I have to do is figure out how to activate multidomain and when your version with "Dynamic rule-based redirects" is ready I will finally be able to bring everything to Shlink! :D**

**Thanks so much Alejandro!**
